docs/development/developing.md

```diff
@@ -53,6 +53,51 @@ Sometimes you might be asked to rebase main into your branch. Please refer to th
 
 If you need help with anything, feel free to reach out and ask for help!
 
+## PyScript Demonstrator
+
+A simple webapp to demonstrate the capabilities of PyScript.
+
+### Getting started
+
+1. If you don't already have Node.js, install it. The official installer for the LTS version of Node is available from [nodejs.org](https://nodejs.org/).
+
+2. If you don't already have Rollup, install it. Rollup can be installed as a global resource using:
+
+
+		$ npm install --global rollup
+
+
+3. Install the demo apps requirements:
+
+
+		$ npm install
+
+
+4. Start the server:
+
+
+		$ npm run dev
+
+
+> This will compile the resources for the app, and start the development server.
+
+
+5. When the compilation completes, it will display something like:
+
+
+			Your application is ready~! 🚀
+
+			- Local:      http://localhost:8080
+			- Network:    Add `--host` to expose
+
+			────────────────── LOGS ──────────────────
+
+
+Once this is visible, open a browser at [http://localhost:8080](http://localhost:8080). This will provide a list of demos that you can run.
+
+### More information
+
+There is a forum PyScript where you can discuss the project or ask questions at [https://community.anaconda.cloud/c/pyscript](https://community.anaconda.cloud/c/pyscript)
```

Comment:
I know this was in the README, but I think this should be expanded to include these other places:

```
* [Discussion board](https://community.anaconda.cloud/c/tech-topics/pyscript)
* [Home Page](https://pyscript.net/)
* [Blog Post](https://engineering.anaconda.com/2022/04/welcome-pyscript.html)
* [Discord Channel](https://discord.gg/BYB2kvyFwm)
```

We have been using discord as the main place for our discussions and ideally we should include it. Probably adding these other ones would be good too 😄

Reply:
@FabioRosado I'll do it during the weekend. I've been busy but I'll make time then.